packages/e2e-test-utils-playwright/src/editor/update-post.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Internal dependencies
+ */
+import type { Editor } from './index';
+
+/**
+ * Updates the post, resolving once the request is complete (once a notice
+ * is displayed).
+ *
+ * @param this
+ */
+export async function updatePost( this: Editor ) {
+	await this.page.click( 'role=button[name="Update"i]' );
```

Comment:
It would be a good idea to narrow down the locator. The "update" is a common label.

```js
await this.page
		.getByRole( 'region', { name: 'Editor top bar' } )
		.getByRole( 'button', { name: 'Update' } )
		.click();
```

Reply:
Totalle makes sense. I just did that in [this commit](https://github.com/WordPress/gutenberg/pull/58676/commits/9fbb7d6153ecd514232351abac1e83318d31e0c9).